google/cloud/tasks/v2beta3/cloudtasks_gapic_v2.yaml

```diff
@@ -0,0 +1,129 @@
+  #   specified resource name formats.
+  #   STATIC_TYPES: the client will use generated types for resource names.
+  #
+  #   page_streaming - Specifies the configuration for paging.
+  #   Describes information for generating a method which transforms a paging
+  #   list RPC into a stream of resources.
+  #   Consists of a request and a response.
+  #   The request specifies request information of the list method. It defines
+  #   which fields match the paging pattern in the request. The request consists
+  #   of a page_size_field and a token_field. The page_size_field is the name of
+  #   the optional field specifying the maximum number of elements to be
+  #   returned in the response. The token_field is the name of the field in the
+  #   request containing the page token.
+  #   The response specifies response information of the list method. It defines
+  #   which fields match the paging pattern in the response. The response
+  #   consists of a token_field and a resources_field. The token_field is the
+  #   name of the field in the response containing the next page token. The
+  #   resources_field is the name of the field in the response containing the
+  #   list of resources belonging to the page.
+  #
+  #   retry_codes_name - Specifies the configuration for retryable codes. The
+  #   name must be defined in interfaces.retry_codes_def.
+  #
+  #   retry_params_name - Specifies the configuration for retry/backoff
+  #   parameters. The name must be defined in interfaces.retry_params_def.
+  #
+  #   field_name_patterns - Maps the field name of the request type to
+  #   entity_name of interfaces.collections.
+  #   Specifies the string pattern that the field must follow.
+  #
+  #   timeout_millis - Specifies the default timeout for a non-retrying call. If
+  #   the call is retrying, refer to retry_params_name instead.
+  methods:
+  - name: ListQueues
+    retry_params_name: default
+  - name: GetQueue
+    retry_params_name: default
+  - name: CreateQueue
+    retry_params_name: default
+  - name: UpdateQueue
+    retry_params_name: default
+  - name: DeleteQueue
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+  - name: PurgeQueue
+    retry_params_name: default
+  - name: PauseQueue
+    retry_params_name: default
+  - name: ResumeQueue
+    retry_params_name: default
+  - name: GetIamPolicy
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+  - name: SetIamPolicy
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+  - name: TestIamPermissions
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 10000
+  - name: ListTasks
+    retry_params_name: default
+    timeout_millis: 10000
+  - name: GetTask
+    retry_params_name: default
+  - name: CreateTask
+    retry_params_name: default
+  - name: DeleteTask
+    retry_params_name: default
```

Comment:
Can you add back in the `retry_codes_name: idempotent` line for DeleteTask? Otherwise it defaults to non-idempotent

Reply:
Also added back some more `retry_codes_name: idempotent` that were deleted by accident, and removed one `retry_codes_name: non-idempotent`.